course05/homework02/freekpaans/linked_list.cc

```diff
@@ -0,0 +1,74 @@
+#include <string>
+#include <numeric>
+
+#include "linked_list.h"
```

Comment:
This include  should be first, then the STL includes. the order of include should always be from the most to least specific, otherwise you won't get a compiler error if anything is missing